service/src/test/java/com/epam/rft/atsy/service/passwordchange/validation/impl/PasswordContainsRuleTest.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.rft.atsy.service.passwordchange.validation.impl;
+
+import com.epam.rft.atsy.service.domain.PasswordChangeDTO;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@RunWith(Parameterized.class)
+public class PasswordContainsRuleTest {
+    @Parameterized.Parameters(name = "{index}: isValid(\"{0}\") should be {1}.")
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][]{
```

Comment:
Please add some more test cases for success cases, and capital letters (success and fail cases also)